Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM python:3.6-alpine
+
+ARG requirements_file=production.txt
+
+ENV PYTHONUNBUFFERED 1
+
+WORKDIR /src
+
+COPY requirements /src/requirements
+
+RUN set -ex \
+    && apk add --update --no-cache --virtual .buildDeps \
+        gcc \
+        libxslt-dev \
+        musl-dev \
+    && pip install --no-cache-dir -r requirements/$requirements_file \
+    && runDeps="$( \
+        scanelf --needed --nobanner --format '%n#p' --recursive /usr/local \
+            | tr ',' '\n' \
+            | sort -u \
+            | awk 'system("[ -e /usr/local/lib/" $1 " ]") == 0 \
+                { next } { print "so:" $1 }' \
+    )" \
+    && apk add --no-cache --virtual .runDeps $runDeps \
+    && apk del .buildDeps \
+    && find /usr/local -depth \
+        \( \
+            \( -type d -a \( -name test -o -name tests \) \) \
+            -o \
+            \( -type f -a \( -name '*.pyc' -o -name '*.pyo' \) \) \
+        \) -exec rm -rf '{}' + \
+    && rm -rf /usr/src/python
+
+COPY . /src/
+
+EXPOSE 8000
+
+CMD ["python", "accesspoc/manage.py", "runserver", "0:8000"]
```

Comment:
Something that you may want to try is to use a multi-stage Docker image. It feels cleaner. There is a good explanation in the following link: https://lekum.org/post/multistage-dockerfile/.

Reply:
Nitpick: `manage.py` is an executable and it has a shebang. It's probably better to use the form:

    CMD ["accesspoc/manage.py", "runserver", "0:8000"]